cpp-ch/local-engine/Storages/SubstraitSource/ReadBufferBuilder.cpp

```diff
@@ -33,6 +33,7 @@
 #include <IO/SeekableReadBuffer.h>
 #include <IO/SharedThreadPools.h>
 #include <IO/SplittableBzip2ReadBuffer.h>
+#include <IO/ParallelReadBuffer.h>
```

Comment:
useless header

Reply:
It is needed otherwise building would fail